dependencies.sh

```diff
@@ -21,11 +21,11 @@ export SPACEMAN_DMM_VERSION=suite-1.8
 # Python version for mapmerge and other tools
 export PYTHON_VERSION=3.9.0
 
-#auxlua repo
-export AUXLUA_REPO=tgstation/auxlua
+#dreamluau repo
+export DREAMLUAU_REPO="y0sh1m4s73r/dreamluau"
```

Comment:
tell mothblocks or whomever headcoder to add this to the org liek we did for auxlua

Reply:
we already have one:
https://github.com/tgstation/auxlua

Reply:
then again I suppose it does have a different branding, we should definitely make a new one